README.md

```diff
@@ -16,6 +16,9 @@ For more information please check:
 
 - [OpenEuropa components](docs/openeuropa-components.md)
 - [How to contribute](docs/how-to-contribute.md)
+- 3rd party open source
+  - Drupal
+    - [Experimental module policies](docs/3rd-party/drupal/experimental-module-policy.md)
```

Comment:
I like this creative solution. This can serve as a placeholder until we have the intermediary pages written.